---
layout: default
title: PAGER
parent: Tools
nav_order: 2
---

![Alt text](/assets/images/pager-logo.png?raw=true "PAGER")

[Web Tool](http://discovery.informatics.uab.edu/PAGER/){:target="_blank"}{: .btn .btn-blue }
[Docs](http://discovery.informatics.uab.edu/PAGER/index.php/pages/help){:target="_blank"}{: .btn .btn-green }
[Ask Questions!](https://uabdatascience.slack.com/archives/C03KRR4RDD5){: .btn }

## About

Pathway and Annotated-list Gene-signature Electronic Repository (PAGER) is an online systems biology tool for constructing and visualizing gene and PAG networks from multiple PAG genesets.

## What is a PAG?
PAG stands for Pathway, Annotated-list and Gene-signature. A PAG geneset represents a set of genes that share a common feature, whether they belong within the same pathway, share common biological processes, are co-differentially expressed under a particular condition of an experiment or clinical sample, or share a common downstream target or function.

## How to cite us

Zongliang Yue, Madhura M. Kshirsagar, Thanh Nguyen, Chayaporn Suphavilai, Michael T. Neylon, Liugen Zhu, Timothy Ratliff, Jake Y. Chen, **PAGER: constructing PAGs and new PAG–PAG relationships for network biology**, _Bioinformatics_, Volume 31, Issue 12, 15 June 2015, Pages i250–i257, <span class="fs-3">[doi](https://doi.org/10.1093/bioinformatics/btv265){:target="_blank"}</span>


# PAGER Web APP

[GitHub](https://github.com/aimed-uab/PAGER-Web-APP){:target="_blank"}{: .btn .btn-purple } 
[Web Tool](http://aimed-lab.shinyapps.io/PAGERwebapp/){:target="_blank"}{: .btn .btn-blue }
[Docs](https://aimed-lab.shinyapps.io/PAGERwebapp/){:target="_blank"}{: .btn .btn-green }
[Ask Questions!](https://uabdatascience.slack.com/archives/C03KRR4RDD5){: .btn }

## About

PAGER Web APP is an interactive online application to perform the gene set enrichment analysis and network interpretation of the functional genomics result using Pathway and Annotated-list Gene-signature Electronic Repository (PAGER). PAGER Web APP provides pre-processed RNA-seq data from UALCAN TCGA data31 and a melanoma drug resistant-sensitive case study from cBioPortal (https://www.cbioportal.org/). We illustrated comparing the enriched pathways from the three leading webservers using application programming interface (API) and how the PAGER Web APP enhances the potential to discover biological insights using network-based computational strategy.

## How to cite us

Yue, Z., Slominski, R., Bharti, S., & Chen, J. Y. (2022), **PAGER Web APP: An Interactive, Online Gene Set and Network Interpretation Tool for Functional Genomics.**, _Frontiers in genetics, 13_, 820361.<span class="fs-3">[doi](https://doi.org/10.3389/fgene.2022.820361){:target="_blank"}</span>



![Alt text](/assets/images/PAGER-CoV-image.png?raw=true "PAGER-CoV")

[GitHub](https://github.com/aimed-uab/PAGER-COV_RUN){:target="_blank"}{: .btn .btn-purple } 
[Web Tool](http://discovery.informatics.uab.edu/PAGER-COV/){:target="_blank"}{: .btn .btn-blue }
[Ask Questions!](https://uabdatascience.slack.com/archives/C03KRR4RDD5){: .btn }

## About

PAGER-CoV (http://discovery.informatics.uab.edu/PAGER-CoV/) is a new web-based database that can help biomedical researchers interpret coronavirus-related functional genomic study results in the context of curated knowledge of host viral infection, inflammatory response, organ damage, and tissue repair. The new database consists of 11 835 PAGs (Pathways, Annotated gene-lists, or Gene signatures) from 33 public data sources. Through the web user interface, users can search by a query gene or a query term and retrieve significantly matched PAGs with all the curated information. Users can navigate from a PAG of interest to other related PAGs through either shared PAG-to-PAG co-membership relationships or PAG-to-PAG regulatory relationships, totaling 19 996 993. Users can also retrieve enriched PAGs from an input list of COVID-19 functional study result genes, customize the search data sources, and export all results for subsequent offline data analysis. In a case study, we performed a gene set enrichment analysis (GSEA) of a COVID-19 RNA-seq data set from the Gene Expression Omnibus database. Compared with the results using the standard PAGER database, PAGER-CoV allows for more sensitive matching of known immune-related gene signatures. We expect PAGER-CoV to be invaluable for biomedical researchers to find molecular biology mechanisms and tailored therapeutics to treat COVID-19 patients.


## How to cite us
Zongliang Yue, Eric Zhang, Clark Xu, Sunny Khurana, Nishant Batra, Son Do Hai Dang, James J Cimino, Jake Y Chen, **PAGER-CoV: a comprehensive collection of pathways, annotated gene-lists and gene signatures for coronavirus disease studies,** _Nucleic Acids Research_, Volume 49, Issue D1, 8 January 2021, Pages D589–D599, <span class="fs-3">[doi](https://doi.org/10.1093/nar/gkaa1094){:target="_blank"}</span>